src/dialogs/notifications/notification-drawer.ts

```diff
@@ -93,28 +93,23 @@ export class HuiNotificationDrawer extends LitElement {
 
     return html`
       <ha-drawer type="modal" open @MDCDrawer:closed=${this._dialogClosed}>
-        <ha-header-bar>
-          <div slot="title">
-            ${this.hass.localize("ui.notification_drawer.title")}
-          </div>
+        <div class="title">
```

Comment:
You're referring to `ha-header-bar`, right?
Can you give an example of what kind of change should impact the notification title?

Reply:
If we decide to change the layout of the header bar, that should also be reflected in the notification drawer. This can be anything, from a font change, to a margin/padding change, to a bugfix.

Reply:
While I still think that's unrealistic (a simple title doesn't really need maintainence), I'll undo the change